hedera-node/src/main/java/com/hedera/services/fees/calculation/UsagePricesProvider.java

```diff
@@ -43,13 +45,34 @@
 	void loadPriceSchedules();
 
 	/**
-	 * Returns the prices in tinyCents that must be paid to
+	 * Returns the prices in a map SubType keys and FeeData values in tinyCents that must be paid to
```

Comment:
I didn't notice for a while either, but the unit of `FeeData` prices is actually 1/1000th of a tinycent.